tests/HUBActionRegistryTests.m

```diff
@@ -58,7 +58,7 @@ - (void)testRegisteringFactoryAndCreatingAction
     HUBActionFactoryMock * const factory = [[HUBActionFactoryMock alloc] initWithActions:@{actionIdentifier.namePart: action}];
     [self.actionRegistry registerActionFactory:factory forNamespace:actionIdentifier.namespacePart];
     
-    XCTAssertEqual([self.actionRegistry createCustomActionForIdentifier:actionIdentifier], action);
+    XCTAssertEqualObjects([self.actionRegistry createCustomActionForIdentifier:actionIdentifier], action);
```

Comment:
Yeah, possibly. I have a problem with `XCTAssertEqual` as it's often misused, and people often aren't asserting what they think. If you *do* want to test instance equality, then using `XCTAssertEqual` just feels wrong as actually tests "more" than equality.